src/qibo/models/dbi/double_bracket.py

```diff
@@ -48,10 +78,16 @@ def __init__(
         self,
         hamiltonian: Hamiltonian,
         mode: DoubleBracketGeneratorType = DoubleBracketGeneratorType.canonical,
+        scheduling: DoubleBracketScheduling = DoubleBracketScheduling.grid_search,
+        cost: DoubleBracketCostFunction = DoubleBracketCostFunction.off_diagonal_norm,
+        state: int = 0,
```

Comment:
```suggestion
        state: int = 0,
```
What is `state` supposed to be here? And why is it an `int`?